REQUIRE

```diff
@@ -1 +1 @@
-julia 0.4
+julia 0.5-
```

Comment:
Yep, that would be the correct REQUIRE sytax. It's `Package LowerBound UpperBound`, where the lower is inclusive and the upper exclusive.